### PR TITLE
Move Commands

### PR DESCRIPTION
**Pull request by TiaSinghania**

### CODE REVIEW

**Review by ethanlshen — APPROVED** [review]
Added javadocs, everything else looks good

File: src/main/java/frc/robot/commands/MoveDirectionCommand.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+   * Specifies distance to travel
+   * @param distance the desired distance to travel
+   * @return returning the object allows for method chaining
+   */
+    public MoveDirectionCommand withDistance (double distance) {
+        m_distance = distance;    
+        return this;
+    }
+
+   /**
```
Comment (noam987) [inline src/main/java/frc/robot/commands/MoveDirectionCommand.java:64]: Be specific in the docs about the difference between heading and direction

**Review by noam987 — CHANGES_REQUESTED** [review]
Clarify the difference between withHeading and withDirection in moveDirectionCommand

File: src/main/java/frc/robot/commands/MoveDirectionCommand.java (hunk 1 above)
Comment (ethanlshen) [reply src/main/java/frc/robot/commands/MoveDirectionCommand.java:64]: Should be better, someone should double check this though

**Review by noam987 — APPROVED** [review]

**Pull request merged**